src/components/JobPost/testPage.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import { Form, Col, Row, Container, Button, Image } from 'react-bootstrap'
+import DateSelector from './DatePicker'
+import uploadImage from '../../media/upload-Image.png'
+// import uploadImage from '../../medi'
+const CompanyProfile = () => {
```

Comment:
The folder should be named by the function: CompanyProfile and the file: index.tsx.

Reply:
That was just page for testing porpose